src/librustdoc/html/static/js/main.js

```diff
@@ -390,23 +390,32 @@ function preLoadCss(cssUrl) {
             if (splitAt !== -1) {
                 const implId = savedHash.slice(0, splitAt);
                 const assocId = savedHash.slice(splitAt + 1);
-                const implElem = document.getElementById(implId);
-                if (implElem && implElem.parentElement.tagName === "SUMMARY" &&
-                    implElem.parentElement.parentElement.tagName === "DETAILS") {
-                    onEachLazy(implElem.parentElement.parentElement.querySelectorAll(
+                const implElems = document.querySelectorAll(
```

Comment:
I think \*think\* that it needs to be "cloned" (like what `onEachLazy` does). However I'm not sure if it's needed here since we don't update the DOM while iterating over it.